packages/rocketchat-language-plugin/server/functions/leave_automatic_channel.js

```diff
@@ -0,0 +1,11 @@
+RocketChat.leave_automatic_channel = function(room_name) {
```

Comment:
I understand that. But why not simply exporting it and importing there? Or why it is even in a separate file?